src/index.html

```diff
@@ -7,8 +7,68 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo">
+        <a href="home-page">
+          <img class="logo_pic"
+          src="images/logo.png"
+          alt="MOYO_Logo">
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="list">
+          <li class="listItem listItem1">
+            <a href="apple" class="navItem navItem1 is-active">
+                APPLE
+              <div class="active"></div>
+            </a>
+
+          </li>
+          <li class="listItem">
+            <a href="samsung" class="navItem navItem2">
+            SAMSUNG
+            </a>
+          </li>
+          <li class="listItem">
+            <a href="smartphones"
+              class="navItem navItem3">
+              SMARTPHONES
+            </a>
+          </li>
```

Comment:
```suggestion
          <li class="listItem">
            <a href="samsung" class="navItem navItem2">
            SAMSUNG
            </a>
          </li>
          
          <li class="listItem">
            <a href="smartphones"
              class="navItem navItem3">
              SMARTPHONES
            </a>
          </li>
```

Add empty line between multiple sibling blocks of html